### ISSUE TITLE
Enable the controller to evaluate the performance of date predictions

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/nlp-sandbox-client#37]
@yy6linda We are now able to generate prediction for date annotation using the controller (previously called Orchestrator). For now at least, prediction files are returned to Synapse. You can find the file that has been successfully pushed to synapse here:

https://www.synapse.org/#!Synapse:syn23512300

Tom needs a command line that makes use of the NLP Sandbox Client to evaluate the performance of a date prediction so that the performance can be returned to Synapse.

## Task

- Identify a command line needed to evaluate the performance of a date annotation predictions
- Test the command line locally using a dummy example
- Share the command line with Tom and give him example files that he can use to test the command line before he integrates it to the controller

**Comment by tschaffter** [comment]
@yy6linda Could you lead this effort this week to provide instruction to Tom on how he can use the client to evaluate the performance of date annotation predictions? Please also work with Tom to create a leaderboard table for date annotation on the staging site according to the columns that we have identifier for this leaderboard. Thanks!

**Comment by yy6linda** [comment]
Task is currently on hold because the date-annotator's output file is not consistent with the example prediction.json file where evaluation script is built on. After talking to Tom and George, we decided to wait until the output format is finalized.

**Comment by tschaffter** [comment]
@yy6linda The information needed for the evaluation is available. Whether the format changes in the future should be a small modification in your evaluation script, so I don't think that it's justified to delay this task. Do you have a proposal to move forward?

**Comment by thomasyu888** [comment]
The evaluation code works but is unable to be tested in the pipeline because the format of the file generated by the date annotator is not the same as the format as the expected prediction file (evaluation code).

**Comment by tschaffter** [comment]
@yy6linda As I mentioned, your evaluation script receives the information needed, even if it's not in the format that your script expects. If this statement is incorrect, please let me know. Otherwise, please add a small snippet that convert the format so that the evaluation so that every tasks downstream (create the leaderboard table, test the submission pipeline, spend some time about what can be improved) can be performed while we work in parallel on defining the final formats.
